python/tvm/relay/op/contrib/forge/forge_passes.py

```diff
@@ -3970,6 +4017,7 @@ def run_forge_compile_passes(relay_module, params=None, inputs=None, target=None
             PadSpecificBatchMatmulShapes(),
             SimplifyVITOnnxAttention(),
             GQABroadcastReshape(),
+            RemoveDenseInputSqueeze(),
```

Comment:
Let's add e2e test to confirm this patter isn't broken in the future: `forge/test/mlir/test_features.py`.

Reply:
Done